Window/TimeField.m

```diff
@@ -8,48 +8,57 @@
 
 #import "TimeField.h"
 
+static NSString *kTimerModeKey = @"timerShowTimeRemaining";
 
-@implementation TimeField
+NSString * formatTimer(long minutes, long seconds, unichar prefix) {
+    return [NSString localizedStringWithFormat:@"%C%lu:%02lu", prefix, minutes, seconds];;
+}
+
+@implementation TimeField {
+    BOOL showTimeRemaining;
+    NSDictionary *fontAttributes;
+}
 
 - (void)awakeFromNib
 {
-    showTimeRemaining = [[NSUserDefaults standardUserDefaults] boolForKey:@"timerShowTimeRemaining"];
+    showTimeRemaining = [[NSUserDefaults standardUserDefaults] boolForKey:kTimerModeKey];
+    if (@available(macOS 10.15, *)) {
```

Comment:
Should still look better than previously on pre-Catalina

Reply:
Hmm, I didn't know that required Catalina. Oh well, better than nothing.